config/routes.rb

```diff
@@ -4,10 +4,8 @@
     resources :categories
   end
   resource :checkout, only: [:create, :update]
-
-  resources :orders, only: [:create]
-  get "checkout/confirmation", to: "checkouts#confirmation"
-  get "checkout", to: "checkouts#show"
+  
+  get 'checkout', to: "checkouts#edit"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.